data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,21 @@
+This project focuses on integrating the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala and wrap flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using it.
```

Comment:
This needs a little story as description. What problem are we trying to solve with this project? How does it help coala and the users of coala? 
We need a motivation for the project (as in why do we want this) and a goal for this.